client/verta/tests/test_cli/test_registry.py

```diff
@@ -251,7 +252,8 @@ def test_create_version_with_custom_modules(self, client, registered_model, crea
                 "type": "torch",
                 "python_version": "2.7.17"
             }
-            model_version.log_artifact("model_api.json", model_api, True, "json")
+            self._log_artifact("model_api.json", model_api, _CommonCommonService.ArtifactTypeEnum.BLOB, 'json',
+                               overwrite=True)
```

Comment:
This reminds me that we don't have any mechanism to log model_api for CLI 🤦

Reply:
I've filed VR-7696 to track this.